packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -27,16 +30,6 @@ function createFetchResult(result: unknown, success = true) {
 	};
 }
 
-export function mockGetMemberships(
```

Comment:
we should write a lint rule that prevents exports from *.test.ts/*.spec.ts files